inc/command/class-command.php

```diff
@@ -1054,7 +1081,270 @@ protected function merge_config( array $config, array $overrides ) : array {
+		$template_file = realpath( __DIR__ . '/../../templates/docslint/.vale.ini' );
+
+		if ( file_exists( $target_file ) ) {
+			$output->writeln( '<error>Documentation linter configuration file already exists at <root>/.vale.ini, not overwriting.</error>' );
+		} else {
+
+			$base_command = sprintf( 'cp "%s" "%s" &> /dev/null', $template_file, $target_file );
+			passthru( $base_command, $return_var );
+
+			if ( $return_var ) {
+				$output->writeln( '<error>Could not create documents linter configuration file, exiting.</error>' );
+
+				return 1;
+			}
+		}
+
+		$output->writeln( '<info>Files have been copied to the root of your project: .markdownlint.jasonc, styles/, and .vale.ini.</info>' );
+		$output->writeln( '<info>Feel free to edit the generated config files to install Style guides or configure as needed.</info>' );
+
+		return 0;
+	}
+
+
+	/**
+	 * Runs Documentation Linting with default config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function lintdocs( InputInterface $input, OutputInterface $output ) {
+		$subcommands = [ 'files', 'markdown', 'style' ];
+		$options = $input->getArgument( 'options' );
+		$subsubcommand = $options[0] ?? null;
+
+		$lint_path = $input->getOption( 'lint-path' ) ?? '.';
+		$lint_path = rtrim( $lint_path, '\\/' );
+
+		if ( ! in_array( $subsubcommand, $subcommands, true ) && ( $subsubcommand !== 'all' ) ) {
+			$output->writeln( sprintf( 'Available sub commands are: %s.', implode( ', ', $subcommands ) ) );
+
+			return 1;
+		}
+
+		// Check path exists
+		if ( ! file_exists( $lint_path ) ) {
+			$output->writeln( sprintf( '<error>Docslint cannot find the path you specified "%s", exiting.</error>', $lint_path ) );
+
+			return 1;
+		}
+
+		if ( $subsubcommand === 'all' ) {
+			foreach ( $subcommands as $subsubcommand ) {
+				call_user_func( [ $this, 'lintdocs_' . $subsubcommand ], $input, $output );
+			}
+		} else {
+			call_user_func( [ $this, 'lintdocs_' . $subsubcommand ], $input, $output );
+		}
+
+		return 0;
+	}
+
+	/**
+	 * Runs the markdown files checker.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function lintdocs_files( InputInterface $input, OutputInterface $output ) : int {
+		$options = $input->getArgument( 'options' );
+		$lint_path = rtrim( $input->getOption( 'lint-path' ), '\\/' );
+		$options = $input->getArgument( 'options' );
+
+		if ( empty( $lint_path ) ) {
+			$output->writeln( sprintf( '<error>Folder not found %s, exiting.</error>', $lint_path ) );
+
+			return 1;
+		}
+
+		$base_command = sprintf(
+			'docker run ' .
+			'--rm ' .
+			'-v %s:/workdir ' .
+			'node:16-alpine sh -c ' .
+			'"node /workdir/vendor/altis/dev-tools-command/templates/docslint/hm-lint-markdown.js /workdir/%s" ',
+			$this->get_root_dir(),
+			$lint_path
+		);
+
+		$output->writeln( sprintf( 'executing: %s.', $base_command ) );
+
+		$return_var = 0;
```

Comment:
I don't think you need to declare this, though some IDEs may be unhappy if you didn't.

Reply:
It is passed by reference into the function, so something needs to be declared.